infra/gcb/build_and_run_coverage.py

```diff
@@ -243,13 +253,28 @@ def get_build_steps(project_dir):
   return build_steps
 
 
+# pylint: disable=missing-function-docstring
 def main():
   if len(sys.argv) != 2:
     usage()
 
+  gcp_project = 'oss-fuzz'
+  gcp_project_base = 'oss-fuzz-base'
   project_dir = sys.argv[1].rstrip(os.path.sep)
   project_name = os.path.basename(project_dir)
-  steps = get_build_steps(project_dir)
+  dockerfile_path = os.path.join(project_dir, 'Dockerfile')
+  project_yaml_path = os.path.join(project_dir, 'project.yaml')
+
+  with open(dockerfile_path) as docker_file:
+    dockerfile_contents = docker_file.readlines()
+
+  with open(project_yaml_path) as project_yaml_file:
+    project_yaml = yaml.safe_load(project_yaml_file)
+
+  workdir = build_project.workdir_from_dockerfile(dockerfile_contents)
```

Comment:
Just do this part get_build_steps? would make it easier for callers, as they won't have to worry about calling workdir_from_dockerfile